src/libs/ajax/fetchAdapter.js

```diff
@@ -46,6 +46,28 @@ async function fetchRequest({ url, method = 'GET', data, params, headers = {}, c
   }
 }
 
+async function fetchMultipartRequest({ url, method = 'POST', data, params, headers = {}, credentials }) {
+  const fullUrl = params ? buildUrlWithParams(url, params) : url
+  const fetchOptions = {
+    method,
+    headers,
+    credentials,
+    body: getRequestBody(data, true),
+    isMultipart: true,
+  }
+  if (headers['Content-Type']) {
```

Comment:
The header for the content type should be handled by the browser when using form multipart.
We would have been able to do this on all requests when multipart is true however consent requires a specific content-type when uploading attachments from the support form.  When we fix that, we can combine the logic in this method into the fetchRequest method.  For now, it's a clean break when dealing with multipart.